src/app/entities/courses/store/actions/courses.actions.ts

```diff
@@ -0,0 +1,18 @@
+import { createAction, props } from '@ngrx/store';
+
+import { Course, CourseData } from '@app/interfaces/entities';
+import { CoursesListData } from '@app/interfaces/parameters';
+
+export const loadCourses = createAction('[Courses] Load', props<CoursesListData>());
+export const loadCoursesPage = createAction('[Courses] Load page', props<CoursesListData>());
+
+export const getCourseById = createAction('[Courses] Get by ID', props<{ id: number }>());
+
+export const deleteCourse = createAction('[Courses] Delete', props<{ id: number }>());
+export const deleteCourseSuccess = createAction('[Courses] Delete success', props<{ id: number }>());
```

Comment:
Just a point for further consideration - how would you separate client's deleteCourse from server's courseDelete.
In our project we use server action like `requestGetCourses`, `requestGetCoursesSuccess`. `requestGetCoursesFail`, and non-api actions don't have the `request` prefix.

Reply:
see: https://blog.nrwl.io/using-ngrx-4-to-manage-state-in-angular-applications-64e7a1f84b7b

Reply:
and other topics from Victor Savkin

Reply:
Ok, thanks, will do renaming of my actions